### DISCUSSION TITLE
Browse Mode maximum character

### DISCUSSION THREAD

**Discussion by harrymkt** [nvaccess/nvda#16723] [Ideas]
Hello.
Please increase the ability to read browser texts (Browse Mode) maximum characters, to 1024 if possible. Currently 250 I believe.
Thanks.

**Comment by CyrilleB79** [comment]
Hi @harrymkt 

Are you talking of the max accepted value for a specific option? Maybe "Maximum number of characters on one line"?
If yes, describing why the current max value is not enough would help.
Thanks.

**Reply by harrymkt** [reply]
Hello.
Because programs like Windows Notepad does that, and I didn't want to scroll through line by line because even the text is one line.

**Comment by seanbudd** [comment]
I've converted this to a discussion. If you'd like to see us triage and allow work to start on the issue, please open a new issue filling out the feature request template in full
